networkx/utils/backends.py

```diff
@@ -95,7 +95,19 @@ class WrappedSparse:
 
 from ..exception import NetworkXNotImplemented
 
-__all__ = ["_dispatchable"]
+__all__ = ["_dispatchable", "backend_config"]
+
+# Get default configuration from environment variables at import time.
+# Default backend config will be initialized from `backend_info` just below.
+backend_config = {
+    "fallback_to_nx": os.environ.get("NETWORKX_FALLBACK_TO_NX", "true").strip().lower()
+    == "true",
+    "automatic_backends": [
```

Comment:
what does `automatic_backends` mean here? like what's its functionality? is it the default backend? 
I don't think I understand all of the backend functionality fully, sorry if I am being repetitive here.